packages/frontend/src/components/ChartDownload.tsx

```diff
@@ -158,40 +198,64 @@ export const ChartDownloadOptions: React.FC<
             </span>
             <Divider />
             <FormGroup label="File format" labelFor="download-type" inline>
-                <HTMLSelect
-                    id="download-type"
-                    value={type}
-                    onChange={(e) =>
-                        setType(e.currentTarget.value as DownloadType)
-                    }
-                    options={Object.values(DownloadType).map(
-                        (downloadType) => ({
-                            value: downloadType,
-                            label: downloadType,
-                        }),
-                    )}
-                />
+                {isTable ? (
+                    <CSVLink
```

Comment:
Cool idea putting the csv download here since the other's don't make much sense. 

@TuringLovesDeathMetal what do you think considering we already have an existing csv download button of the explore table?
<img width="1513" alt="Screenshot 2022-01-31 at 14 59 27" src="https://user-images.githubusercontent.com/11660098/151817021-e5a21d4f-74d7-4ecb-8612-3c3360ad1cec.png">